src/scaffoldmaker/meshtypes/meshtype_3d_stellate1.py

```diff
@@ -69,16 +62,19 @@ def checkOptions(options):
         for key in [
             'Element length along arm',
             'Element width across arm',
-            'Element thickness',
-            'Extent of indentation around centre']:
+            'Element thickness']:
             if options[key] < 0:
                 options[key] = 0.1
 
-        if options['Number of elements in short arms'] < 2:
-            options['Number of elements in short arms'] = 2
-
-        # if options['Number of arms'] < 3 or options['Number of arms'] > 4:
-        #     options['Number of arms'] = 3
+        key = 'Number of elements in all arms (e.g. 4,2,2)'
+        if len(options[key]) < 3:
+            x = [2]*(3 - len(options[key]))
+            options[key].append(x)
```

Comment:
This doesn't actually work; line 2 (x = ...) makes a list, line 3 appends it to the list i.e. adding a last element that is itself a list. You could change the last line to options[key] += x
However, I would do this differently to make it ready for variable numbers of arms, and to avoid the issue where if you have 2 missing arms, the code may make the last 2 values references to the same value 2:
```
armCountsKey = 'Numbers of elements along arms'
armCount = 3  # to allow for it being a parameter later
countsLen = len(options[armCountsKey]
if countsLen < armCount:
    options[armCountsKey] += [ 2 for i in range(armCount - countsLen) ]
elif countsLen > armCount:
    options[armCountsKey] = options[armCountsKey][:armCount]
'''